x/ibc/04-channel/keeper/timeout.go

```diff
@@ -99,13 +106,6 @@ func (k Keeper) TimeoutPacket(
 		return nil, sdkerrors.Wrap(types.ErrInvalidPacket, "packet verification failed")
 	}
 
-	k.deletePacketCommitment(ctx, packet.GetSourcePort(), packet.GetSourceChannel(), packet.GetSequence())
```

Comment:
why don't we delete the packet commitment?

Reply:
^^

Reply:
Moved to `TimeoutExecuted`